src/cards/CameraCard.js

```diff
@@ -0,0 +1,39 @@
+import {AbstractCard} from "./AbstractCard";
+
+/**
+ * Camera Card Class
+ *
+ * Used to create a card for controlling an entity of the camera domain.
+ *
+ * @class
+ * @extends AbstractCard
+ */
+class CameraCard extends AbstractCard {
+  /**
+   * Default options of the card.
+   *
+   * @type {cameraCardOptions}
+   * @private
+   */
+  #defaultOptions = {
+    type: "custom:webrtc-camera",
+    icon: undefined,
```

Comment:
is `icon` necessary?

Reply:
I don't think so.
Probably a left-over from a copy/paste action.
Will be removed.